plugins/include/cstrike.inc

```diff
@@ -209,6 +209,20 @@ native cs_get_user_buyzone(index);
  */
 native cs_get_user_hasprim(index);
 
+/**
+ * Returns if the client has a weapon in the specified inventory slot.
+ *
+ * @param index     Client index
+ * @param slot      Slot to check. See CS_WEAPONSLOT_* in cstrike_const.inc for a list of possible values.
```

Comment:
I am not against listing them, but I don't think it's really needed. Would probably save people trying to use the native around 5 seconds of searching.

Reply:
5 seconds here, 5 seconds there, and you have 5 minutes in the end. It's better to have them listed in the description rather than having to search for them. Since it's already done for other new natives, it should be done here as well.

Reply:
"for other natives" likely only https://github.com/alliedmodders/amxmodx/pull/604
Most if not all other natives are not listing them:
Examples:
cs_[g/s]et_user_bpammo: "For a list of possible weapon ids see the CSW_* constants in  amxconst.inc"
cs_set_user_team: "For a list of valid team ids see the CsTeams enum, and for a list of valid internal model ids see the CsInternalModel enum."
cs_[g/s]et_user_armor: "For a list of possible armor types see the CsArmorType enum."
cs_[g/s]et_user_zoom: "For a list of possible zoom types see the zoom type enum above (CS_*_ZOOM constants)."
cs_get_item_id: " * @param classid       If item is a weapon, variable to store the associated  weapon class id in (CS_WEAPONCLASS_* constants)"
cs_get_weapon_info: "* @param type          Info type, see CS_WEAPONINFO_* constants"
set_pev: "Sets entvar data for an entity.  Use the pev_* enum from fakemeta_const.inc for reference."

And the list can go on and on. Feel free to make a PR and address all of them. 
This could be a problem for bigger lists of constants, you are going to have big chunks of duplicated text that's going to pollute the documentation(for example copying all the CSW_* constants in every native that takes a weapon index as parameter). 
For lists with few items that shouldn't be a big deal tho. 

IMO it's already good as it is right now. The reader is clearly told what the possible values for the parameter are and where to find them.

Reply:
It's fine to add them, but ultimately I would prefer to avoid it for at least two reasons:
* Don't Repeat Yourself; It should have only one occurrence of the constants for the sake of maintainability.
* We need to improve the online doc and we should encourage people to use it. For example, https://www.amxmodx.org/api/amxmodx/find_player_ex ; the `FindPlayerFlags` tag should be linked to the page where it's located. SourceMod has already this kind of system, like for example: https://sm.alliedmods.net/new-api/admin/FindFlagByName

Reply:
Yeah, linking the constants is the right way to do it. No repeating and no wasting time.